### PR TITLE
Weekly update for 2015-08-07

### PR DESCRIPTION
**Pull request by diagramatics**
Also going to port this to the new.nodejs.org ASAP.

Notes:
- @yosuke-furukawa's [iojs-new-features](https://github.com/yosuke-furukawa/iojs-new-features/pull/1) link. If it's merged to master and the permalink's permanent I can put it in if necessary. I have no technical expertise in the topics and only summarized what he wrote so a double check if the things I wrote make sense would be great.
- #124 links applied again.
- With the summary of 3.0.0 I'm not sure if Notable Changes is still necessary or if we should just redirect readers to CHANGELOG.md instead.
- The v3.0.0 topic is quite lengthy so I dropped Notable Changes and Known Issues from H3 to H4.

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
@diagramatics Thank you for updates!
I have merged my repo to the master. https://github.com/yosuke-furukawa/iojs-new-features

File: weekly-updates/weekly-update.2015-08-07.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+### io.js and Node.js News — August 7th
+io.js v3.0 and the new goodness, breaking changes, and our ongoing call for an LTS build V8 maintainer.
+
+### io.js v3.0 Release
+
+This week io.js [v3.0.0](https://iojs.org/dist/v3.0.0/) is released! Since this is a major version bump, new features have been introduced to io.js. These are:
```
Comment (Trott) [inline weekly-updates/weekly-update.2015-08-07.md]: Nit: add comma after `week`

**Comment by diagramatics** [comment]
Thanks @Trott, fixed.

I also added the repo link to the end of the section.

**Comment by yosuke-furukawa** [comment]
Mostly LGTM

**Comment by diagramatics** [comment]
Seeing how the new.nodejs.org PR is merged, I suppose that's two LGTM.

**Pull request merged**